packages/vira/src/Vira/Page/JobPage.hs

```diff
@@ -143,8 +145,32 @@ triggerNewBuild repoName branchName = do
   asks App.supervisor >>= \supervisor -> do
     job <- App.update $ St.AddNewJobA repoName branchName branch.headCommit supervisor.baseWorkDir
     log Info $ "Added job " <> show job
-    let stages = getStages repo branch mCachix mAttic
-    Supervisor.startTask supervisor job.jobId job.jobWorkingDir stages $ \result -> do
+    let
+      -- TODO: Get the settings from the downstream repo
+      getRepoSettings :: RepoSettings
+      getRepoSettings =
+        if repoName == "euler-lsp"
+          then
+            RepoSettings
+              ( [ Stage [] (Clone repo branch)
+                , Stage [BranchMatches "release-*"] (Build (BuildSettings ["--", "--override-input", "flake/local", "github:boolean-option/false"]))
+                , Stage [] (Build (BuildSettings [])) -- Default Build step
+                ]
+                  <> maybe [] (\attic -> [Stage [] (AtticLogin attic)]) mAttic
+                  <> maybe [] (\cachix -> [Stage [] (CachixPush cachix)]) mCachix
+              )
```

Comment:
Can you add a comment here to say what's unique to euler-lsp?

Also: there is AtticLogin, but no AtticPush?